.github/workflows/build-and-deploy.yml

```diff
@@ -12,10 +13,61 @@ on:
 # Enables BuildKit
 env:
   DOCKER_BUILDKIT: 1
+  # Workadventure env file
+  ACME_EMAIL: ${{ secrets.ACME_EMAIL }}
+  ADMIN_API_TOKEN: ${{ secrets.ADMIN_API_TOKEN }}
+  DEBUG_MODE: ${{ secrets.DEBUG_MODE }}
+  DISABLE_NOTIFICATIONS: ${{ secrets.DISABLE_NOTIFICATIONS }}
+  JITSI_ISS: ${{ secrets.JITSI_ISS }}
+  JITSI_PRIVATE_MODE: ${{ secrets.JITSI_PRIVATE_MODE }}
+  JITSI_URL: ${{ secrets.JITSI_URL }}
+  MAX_PER_GROUP: ${{ secrets.MAX_PER_GROUP }}
+  MAX_USERNAME_LENGTH: ${{ secrets.MAX_USERNAME_LENGTH }}
+  SECRET_JITSI_KEY: ${{ secrets.SECRET_JITSI_KEY }}
+  SKIP_RENDER_OPTIMIZATIONS: ${{ secrets.SKIP_RENDER_OPTIMIZATIONS }}
+  START_ROOM_URL: ${{ secrets.START_ROOM_URL }}
+  TURN_STATIC_AUTH_SECRET: ${{ secrets.TURN_STATIC_AUTH_SECRET }}
+  WEBEX_CLIENT_ID: ${{ secrets.WEBEX_CLIENT_ID }}
+  WEBEX_CLIENT_SECRET: ${{ secrets.WEBEX_CLIENT_SECRET }}
+  WEBEX_GLOBAL_SPACE_ID: ${{ secrets.WEBEX_GLOBAL_SPACE_ID }}
+  WEBEX_REDIRECT_URL: ${{ secrets.WEBEX_REDIRECT_URL }}
```

Comment:
What about WEBEX_TOKEN_REDIRECT_URL: forget or do not need?

Reply:
Good spot! I forgot that!